adapters/python/sinks/metadata_json.py

```diff
@@ -201,6 +228,32 @@ def get_tag_location(frame: VideoFrame):
     return value.as_string()
 
 
+def opt_config_metadata_format(
+    env_var_name: str = 'METADATA_JSON_FORMAT',
+) -> MetadataJsonFormat:
+    """Parse the metadata format from an environment variable.
+
+    Args:
```

Comment:
Please, use reStructuredText style for comments 
https://stackoverflow.com/questions/3898572/what-are-the-most-common-python-docstring-formats